File: src/main/java/ch/java8/workshop/java/Address.java
package ch.java8.workshop.java;

public class Address {
	
	private String location;
	
	private String country;
	
	public Address() {
		super();
	}

	public Address(String country) {
		super();
		this.country = country;
	}

	public Address(String location, String country) {
		super();
		this.location = location;
		this.country = country;
	}

	public String getLocation() {
		return location;
	}

	public void setLocation(String location) {
		this.location = location;
	}

	public String getCountry() {
		return country;
	}

	public void setCountry(String country) {
		this.country = country;
	}

	public String toString() {
		return "[" + location + " " + country +"]";
	}
}

File: src/main/java/ch/java8/workshop/cs/Address.cs
using System;
public class Address {
	
	public string location { get; set; }
	
	public string country { get; set; }
	
	public Address(string country) {
		this.country = country;
	}

	public Address(string location, string country) {
		this.location = location;
		this.country = country;
	}
	
	public override string ToString(){
        return String.Format("{0}   |   {1}", location, country);
    }
}